src/v2/Apps/Conversation/Components/__tests__/Conversation.jest.tsx

```diff
@@ -55,28 +64,14 @@ describe("Conversation", () => {
       }
     })
 
-    it("shows the buyer guarantee message", () => {
```

Comment:
some pointless test separated in different tests, grouped them in the test below